spec/acceptance/apt_key_provider_spec.rb

```diff
@@ -42,7 +49,7 @@
 
           apply_manifest(pp, :catch_failures => true)
           apply_manifest(pp, :catch_changes => true)
-          shell("apt-key list | grep #{PUPPETLABS_GPG_KEY_ID}")
+          shell("apt-key adv --list-keys --with-colons --fingerprint | grep #{PUPPETLABS_GPG_KEY_FINGERPRINT}")
```

Comment:
In the provider you created a `cli_args` so we didn't have to repeat ourselves all over the place. Could you repeat that for the tests too?